app/backend/src/tests/test_fixtures.py

```diff
@@ -919,7 +919,7 @@ def __getattr__(self, attr):
             try:
                 return self.kwargs[attr]
             except KeyError:
-                raise AttributeError(f"'{self.__class__.__name__}' object has no attribute '{attr}'")
+                raise AttributeError(f"'{self.__class__.__name__}' object has no attribute '{attr}'") from None
```

Comment:
I'm not sure what this expression does?

Reply:
To have a traceback of exceptions you can write `raise Exception from other_exc`. B904 states that you should be explicit if you don't want that https://docs.astral.sh/ruff/rules/raise-without-from-inside-except/

Reply:
Thanks!